telemetry/definitions/commonDefinitions.json

```
@@ -850,15 +898,38 @@
             "name": "apprunner_viewServiceLogs",
             "description": "View the App Runner service logs (the logs produced by App Runner)"
         },
+        {
+            "name": "auth_addConnection",
+            "description": "Called when user clicks on add New Connections from any of entry points. fields captures snapshot of user state upon reaching a result",
+            "metadata": [
+                { "type": "source"},
+                { "type": "tab", "required": false},
+                { "type": "unlockedTabs", "required": false},
+                { "type": "numberOfConnections", "required": false},
+                { "type": "credentialSourceId", "required": false},
+                { "type": "invalidInputFields", "required": false},
+                { "type": "iamIdentityCenterRegion", "required": false },
+                { "type": "reason", "required": false},
+                { "type": "result"}
+            ]
+        },
+        {
+            "name": "auth_switchBetweenTheRoles",
```

Could be less verbose without losing clarity.

```suggestion
            "name": "auth_switchRoles",
```